src/main/java/org/folio/coursereserves/util/Util.java

```diff
@@ -1,7 +1,6 @@
 package org.folio.coursereserves.util;
 
-import io.vertx.core.logging.Logger;
-
+import org.apache.logging.log4j.Logger;
```

Comment:
Replacing `getLocalizedMessage` with `getMessage` a few lines below is more consistent with the other changes.